src/foam/java/refinements.js

```diff
@@ -10,7 +10,9 @@ foam.CLASS({
     {
       class: 'String',
       name: 'javaType',
-      factory: function() { return this.of; }
+      factory: function() {
```

Comment:
Why this change?

Reply:
Fixed: https://github.com/foam-framework/foam2/pull/1239